### DISCUSSION TITLE
配合smartdns单独禁用国外的ipv6地址

### DISCUSSION THREAD

**Discussion by a904055262** [Openwrt-Passwall/openwrt-passwall2#120] [General]
smartdns开启第二组DNS，当作远程DNS，设置如图
![网页捕获_26-5-2022_125142_192 168 6 2](https://user-images.githubusercontent.com/3348789/170418556-737e0563-fd9e-4143-be17-eeb9d88c57ce.jpeg)
passwall设置如图，国内的不动，就让passwall用运营商的dns解析ip
![网页捕获_26-5-2022_12525_192 168 6 2](https://user-images.githubusercontent.com/3348789/170418590-39cd310d-83c6-4d3a-bd84-abf3d99c959e.jpeg)

这样既能保留国内ipv6的dns解析，又能屏蔽国外的ipv6，从而不影响科学上网，

不知道这样设置对不对

**Comment by xlyeah** [comment]
我也想知道这个问题，和楼主的配置方法一样，但是不生效 nslookup ipv6.google.com 还是返回了ipv6的地址
只有把UseIp改成UseIp4才可以。
因为梯子不支持ipv6 所以有概率打不开页面；

（ps 我是把跳过address nameserver ipset soa 双栈优选 都开了   第一服务器也开了 作为dnsmasq上游

**Comment by handsomeji** [comment]
最新的源码已经是有两组DNS策略可设置
